docs/index.md

```diff
@@ -1,20 +1,18 @@
 ## Autonomous Economic Agent Framework
 
-The AEA framework provides the tools for creating autonomous economic agents. Autonomous Economic Agents (AEAs) work continuously for your benefit without you having to do anything more than writing them and starting them up.
+The AEA framework provides the tools for creating Autonomous Economic Agents (AEA). AEAs work continuously for your benefit without you having to do anything more than writing them and starting them up.
 
-AEAs act independently of constant input and autonomously develop new capabilities. Their goal is to create economic gain for you, their owner. AEAs have a wide range of application areas. Check out the <a href="/aea/gym-plugin/" target=_blank>demo section</a> for some examples.
+AEAs act independently of constant input and autonomously expand their capabilities by learning new skills. Their goal is to create economic value for you, their owner. AEAs have a wide range of application areas. Check out the <a href="/car-park">Demos section</a> from the left menu for some examples.
```

Comment:
I prefer the previous sentence, if the new one, then it would have to be `acquire new skills` as skills aren't learned by the agents.